WalletWasabi.Fluent/ViewModels/HelpAndSupport/AboutViewModel.cs

```diff
@@ -26,6 +27,72 @@ public partial class AboutViewModel : RoutableViewModel
 	{
 		public AboutViewModel()
 		{
+			Links = new List<ViewModelBase>()
+			{
+				new LinkViewModel()
+				{
+					Link = DocsLink,
+					Description = "Documentation",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = SourceCodeLink,
+					Description = "Source Code (Github)",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = ClearnetLink,
+					Description = "Website (Clearnet)",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = TorLink,
+					Description = "Website (Tor)",
+					IsClickable = false
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = StatusPageLink,
+					Description = "Coordinator Status Page",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = UserSupportLink,
+					Description = "User Support",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = BugReportLink,
+					Description = "Bug Reporting",
+					IsClickable = true
+				},
+				new SeparatorViewModel(),
+				new LinkViewModel()
+				{
+					Link = FAQLink,
+					Description = "FAQs",
+					IsClickable = true
+				},
+			};
+
+			License = new()
```

Comment:
Reference to this https://github.com/zkSNACKs/WalletWasabi/discussions/5639 I would not use the sort version `new` here. Because it is not straightforward what is the type of the `License`.

Reply:
Agree, anyway rider shows type anyway so wasn't that obvious when looking at code in IDE

Reply:
Pushed changes